_posts/2015-02-23-taskcluster-update.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Taskcluster Release Part 1 : Gecko"
+description: "TC Update 1"
+category: mozilla, ci
+tags: [ci, docker, mozilla]
+---
+
+It's been awhile since my last blog post about taskcluster and I wanted to give an update...
+
+## Taskcluster + Gecko
+
+Taskcluster is running by default on
+
+  - [try](https://treeherder.allizom.org/#/jobs?repo=try)
+  - [b2g-inbound](https://treeherder.allizom.org/#/jobs?repo=b2g-inbound)
+  - [mozilla-inbound](https://treeherder.allizom.org/#/jobs?repo=mozilla-inbound)
+  - [fx-team](https://treeherder.allizom.org/#/jobs?repo=fx-team)
+  - [mozilla-central](https://treeherder.allizom.org/#/jobs?repo=mozilla-central)
+
+
+You will see both buildbot run and taskcluster run jobs the "TC" jobs are
+prefixed accordingly so you can tell the difference.
+
+This is the last big step to enabling TC as the default CI for many mozilla
+projects also new branches are a simple config away.
+
+Why is this a great thing? Just about everything is [in](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/mach_commands.py) [the](https://dxr.mozilla.org/mozilla-central/source/testing/docker) [tree](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/tasks/builds).
+
+This means you can easily add new builds/tests and immediately push them
+to try for testing (see the configs for [try](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/tasks/branches/try/job_flags.yml)
+
+Results currently show up only on [staging treeherder](https://treeherder.allizom.org)
+aside from the ease of adding tests there are a few other key benefits
```

Comment:
'Results are shown on [...] only.' I would create a new paragraph that sets up the list of key benefits and retool the sentence -- 'Adding new tests is easier than ever but the improvements don't stop there. Other key benefits on linux include:'